### DISCUSSION TITLE
Facebook Login Error: 'Feature Not Available'

### DISCUSSION THREAD

**Discussion by rodrigoGA** [capawesome-team/capacitor-firebase#373] [Q&A]
I am facing some issues while developing the login feature in my application. Specifically, I am trying to implement the Facebook login feature, but I keep receiving an error message. I have attached a screenshot of the error message for reference.

Interestingly, the login feature works perfectly fine on the web version of the application, but on Android, no matter how much I modify the configuration, the error message persists. I am using Angular Fire and I have tried setting skipNativeAuth to both true and false, but unfortunately, I keep getting the same error message.

I would really appreciate it if anyone could provide me with some guidance on how to resolve this issue. Thank you in advance for your help!

The translation of the error message is similar to the following:

"Feature not available. At this time, Facebook login is not available because we are updating other details of the app. Please try again later."

![1](https://user-images.githubusercontent.com/2334523/235326509-429066dd-7eac-4c64-8e3f-4d5fdf11d3e5.jpeg)

**Comment by robingenz** [comment]
I've never seen this error message before and the last time I configured Facebook Login was a while ago. I plan to publish a blog post in the next few months on how to set up each login provider. But unfortunately I can't think of anything at the moment. Maybe someone else in the community can help you out.
